Traceback (most recent call last):
  File "2BeamDynamics.py", line 159, in <module>
    CorrectionFactor = np.loadtxt(CFD, skiprows=1)[-1]
  File "/home/mv15885/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 917, in loadtxt
    fh = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/home/mv15885/anaconda2/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/mv15885/anaconda2/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 616, in open
    raise IOError("%s not found." % path)
IOError: /home/mv15885/Project/1DHopping1.8/CorrectionFactorsping1.8 not found.
